src/blog/_posts/2023-06-08-survey-results.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We released a user survey earlier this year to learn about who our users are, how they use Cadence, and how we can help them. After collecting the feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable and it helps us shape our roadmap for the future.
+
+
+Here are some highlights in text and you can check out the visuals to get more details:
+
+![using.png](../../shared/img/2023-survey-results/using.png)
+
+![job_role.png](../../shared/img/2023-survey-results/job_role.png)
+
+Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating, or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale, and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace for guidance.
```

Comment:
Is this is a survey skew result? Nothing wrong with it, but I'm guessing we've selected for folks already using it by surveying from folks in the cadence slack and similar channels

Reply:
Yeah most people who follow us are engineers but the survey was shared from Slack, Blog and LinkedIn with multiple reshares by other people.